.editorconfig

```diff
@@ -0,0 +1,38 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
This probably belongs in .gitignore if this is sopecific to your user environemnt! Please either add this to a gitignore, or delete this file